react-restaurant/src/Food.js

```diff
@@ -7,25 +7,28 @@ class Food extends React.Component {
     }
 
     componentDidMount() {
-        axios.get(`https://entree-f18.herokuapp.com/v1/menu/12`)
-            .then(response => {
-                console.log(response.data);
-                console.log(response.data.menu_items);
-                console.log(response.data.menu_items[0].description);
-                this.setState({ content: response.data.menu_items });
-                console.log(this.state.content)
-            })
+        if (localStorage.getItem(this.props.menuSection) === null) {
+            axios.get(`https://entree-f18.herokuapp.com/v1/menu/12`)
+                .then(response => {
+                    this.setState({ content: response.data.menu_items });
+                    
+                    // SetItem(key(each menu section), value)
+                    window.localStorage.setItem(this.props.menuSection, JSON.stringify(this.state.content));
+                })
+        }else{
+            this.setState({ content: JSON.parse(localStorage.getItem(this.props.menuSection))})
+        }
     }
 
     render() {
         return (
             <div className='card'>
                 <div className='card-body'>
-                    {this.state.content.map(item =>
+                    {this.state.content.map((item, idx) =>
                         <>
```

Comment:
Yes, from what I understand it's the same thing as wrapping it in a `<div>` tag. Just tried to remove it locally and it didn't compile. Maybe there is another way around it? Thanks.

Reply:
Odd, I've never seen that before but it might be a valid element in JSX. I would recommend sticking with a `<div>` or equivalent containing element for clarity, but that's just my approach 👍